packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -2133,9 +2305,10 @@ class RoundRangeSliderTickMarkShape extends RangeSliderTickMarkShape {
 
     // The tick marks are tiny circles that are the same height as the track.
     final double tickMarkRadius = getPreferredSize(
-      isEnabled: isEnabled,
-      sliderTheme: sliderTheme,
-    ).width / 2;
+          isEnabled: isEnabled,
+          sliderTheme: sliderTheme,
+        ).width /
+        2;
```

Comment:
There's a ton of gratuitous reformatting in this file. This has to be the worst of it.

Reply:
Done